### ISSUE TITLE
Temperature results do not display with VTK file

### ISSUE THREAD

**Issue by wigging** [InsightSoftwareConsortium/itk-jupyter-widgets#168]
I would like to use itk-jupyter-widgets to view battery simulation results in a JupyterLab notebook. Previously I used [K3D](https://github.com/K3D-tools/K3D-jupyter) to view the VTK file (see example below).

```python
import k3d
import vtk

filename = 'case1.vtk'

reader = vtk.vtkUnstructuredGridReader()
reader.SetFileName(filename)
reader.Update()

geometryFilter = vtk.vtkGeometryFilter()
geometryFilter.SetInputData(reader.GetOutput())
geometryFilter.Update()

plot = k3d.plot()
thermal = k3d.vtk_poly_data(geometryFilter.GetOutput(), color_attribute=('Battery_Temperature', 296, 316))
plot += thermal
plot.display()
```

My attempt with the itk-jupyter-widget is shown below.

```python
import vtk
from itkwidgets import view

file_name = 'case1.vtk'

reader = vtk.vtkUnstructuredGridReader()
reader.SetFileName(file_name)
reader.Update()

geometryFilter = vtk.vtkGeometryFilter()
geometryFilter.SetInputData(reader.GetOutput())
geometryFilter.Update()
case1 = geometryFilter.GetOutput()

view(geometries=case1)
```

The itk widget will display the VTK file in the notebook but the temperature gradient from the simulation results is not visible. In K3D, the temperature results were defined with `color_attribute=('Battery_Temperature', 296, 316)` but I do not see how to accomplish this with the itk widget.

**Comment by thewtex** [comment]
@wigging is it possible to share the *case1.vtk* to reproduce the behavior?

**Comment by wigging** [comment]
@thewtex Here is the `case1.vtk` file that I am working with.

[case1.vtk.zip](https://github.com/InsightSoftwareConsortium/itk-jupyter-widgets/files/3413006/case1.vtk.zip)

**Comment by thewtex** [comment]
@wigging thanks!

**Comment by wigging** [comment]
@thewtex Thank you for merging this feature. Do you know when it will be available as part of the pip package? Current version on pip is 0.17.1

**Comment by thewtex** [comment]
@wigging it is now available in 0.18.0 :tada:
